### ISSUE TITLE
Bug: Alert screen reader accessibility

### ISSUE THREAD

**Issue by cliu02** [bloom-housing/ui-seeds#114]
Flagging for @jaredcwhite @ColinBuyck @emilyjablonski, let me know if you have questions! 

**Issue:** 

DAHLIA's most recent audit by the SFDDS accessibility consultant uncovered that Create Account page error messages are not announced by mobile screen readers. Mobile screen reader users are not notified of any missing or incorrect information and how to resolve the issue. As a result they are unable to correct the issue and proceed with their task. 

The [Alert component](https://github.com/bloom-housing/ui-seeds/blob/main/src/blocks/Alert.tsx) and the [CommonMessage component](https://github.com/bloom-housing/ui-seeds/blob/main/src/blocks/shared/CommonMessage.tsx) that it extends should support being present in the DOM even when there are no children elements.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/alert_role:

As with all other live regions, alerts will only be announced when the content of the element with role="alert" is updated. Make sure that the element with the role is present in the page's markup first - this will "prime" the browser and screen reader to keep watching the element for changes. After this, any changes to the content will be announced. Do not try to dynamically add/generate an element with role="alert" that is already populated with the alert message you want announced - this generally does not lead to an announcement, as it is not a content change.

**To replicate:**

1. On https://dahlia-full.herokuapp.com/create-account, submit new account information and data that is either incorrect or missing. 
2. The mobile screen reader does not announce the error message that is triggered after submission.

**Comment by cade-exygy** [comment]
@ColinBuyck @emilyjablonski Hey all, I wanted to ping you both as I am about to take a look. Should I find a fix and make a PR or do you all have preference / capacity to take this one?

**Comment by ludtkemorgan** [comment]
@cade-exygy if you have capacity to address this feel free to create a PR. @ColinBuyck and @emilyjablonski are out the next few days, but if you create a PR can you add them as reviewers?
